examples/undocumented/python/graphical/kernel_ridge_regression_sinc.py

```diff
@@ -1,31 +1,30 @@
-from pylab import figure,pcolor,scatter,contour,colorbar,show,subplot,plot,legend,connect
-from shogun import *
+import matplotlib.pyplot as plt
+import shogun as sg
 import util
 
-util.set_title('KernelRidgeRegression on Sine')
+plt.figure('KernelRidgeRegression on Sine')
 
+X, Y = util.get_sinedata()
+width = 1
 
-X, Y=util.get_sinedata()
-width=1
-
-feat=RealFeatures(X)
-lab=RegressionLabels(Y.flatten())
-gk=GaussianKernel(feat, feat, width)
-krr=KernelRidgeRegression()
-krr.set_labels(lab)
-krr.set_kernel(gk)
-krr.set_tau(1e-6)
+feat = sg.features(X)
+lab = sg.labels(Y.flatten())
+gk = sg.kernel('GaussianKernel', log_width=width)
+gk.init(feat, feat)
+krr = sg.machine('KernelRidgeRegression')
```

Comment:
could you use python `**kwargs`, i.e.
`krr = sg.machine('KernelRidgeRregression", labels=lab, kernel=gk, tau=1e-3)` (and for all the other instantiations as well). Makes everything look more like Python :)